assets/src/edit-story/components/library/panes/pageLayouts/pageLayout.js

```diff
@@ -18,32 +18,44 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import { useRef, useState, useCallback } from 'react';
+import { useState, useCallback, useRef } from 'react';
 import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
+import { useFocusOut, useKeyDownEffect } from '../../../../../design-system';
 import { PageSizePropType } from '../../../../types';
 import { PreviewPage, PreviewErrorBoundary } from '../../../previewPage';
-import useFocusOut from '../../../../utils/useFocusOut';
 import { STORY_ANIMATION_STATE } from '../../../../../animation';
 import { KEYBOARD_USER_SELECTOR } from '../../../../utils/keyboardOnlyOutline';
+import useRovingTabIndex from '../../../../utils/useRovingTabIndex';
```

Comment:
I've been noticing that we have a ton of `../` all around our codebase. Maybe in the future we can add some aliases so it would be easier to figure out where these are coming 🤔